docs/docs/cmd/spo/file/file-get.md

```diff
@@ -82,3 +85,211 @@ Save file with server-relative url _/sites/project-x/documents/Test1.docx_ locat
 ```sh
 m365 spo file get --webUrl https://contoso.sharepoint.com/sites/project-x --url '/sites/project-x/documents/Test1.docx' --asFile --path /Users/user/documents/SavedAsTest1.docx
 ```
+
+Get file properties for file with id (UniqueId) _b2307a39-e878-458b-bc90-03bc578531d6_ located in site _https://contoso.sharepoint.com/sites/project-x_ with permissions
```

Comment:
Similar comment we could add use more general descriptions for the examples and along the way refactor the current once 👍